### DISCUSSION TITLE
0094. 二叉树的中序遍历

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#191] [Comments]
# 0094. 二叉树的中序遍历

[https://algo.itcharge.cn/solutions/0001-0099/binary-tree-inorder-traversal/](https://algo.itcharge.cn/solutions/0001-0099/binary-tree-inorder-traversal/)

**Comment by jarrett-au** [comment]
抓个虫，这里的描述应该还是pre-order的没改过来：
> 二叉树的前序遍历显式栈实现步骤如下：
判断二叉树是否为空，为空则直接返回。
初始化维护一个栈，将根节点入栈。
当栈不为空时：
弹出栈顶元素 node，并访问该元素。
如果 node 的右子树不为空，则将 node 的右子树入栈。
如果 node 的左子树不为空，则将 node 的左子树入栈。

**Reply by itcharge** [reply]
感谢指正，已修正。
